src/services/jwt.service.js

```
@@ -0,0 +1,38 @@
+import jwt from 'jsonwebtoken';
```

Consider adding a module-level check that required env variables exist (JWT_KEY and JWT_REFRESH_KEY). If these are missing `jwt.sign`/`jwt.verify` will throw at runtime; failing fast with a clear message makes debugging easier.